### PR TITLE
extend example to allow user to specify URI

### PR DESCRIPTION
**Pull request by ultrasaurus**
user enters URI as suggested in feedback to: https://github.com/sgodwincs/rtsp-rs/pull/33
currently only supports URL with ipAddr

Many thanks to my co-author @denniscollective who is helping me on my Rust journey

### CODE REVIEW

**Comment by sgodwincs** [comment]
Can you update the request builder to use the URI given by the user?

Other than that, LGTM. Thanks!

File: examples/client.rs [hunk 1]
```diff
@@ -9,11 +9,35 @@ use rtsp::client::Client;
 use rtsp::method::Method;
 use rtsp::request::Request;
 use rtsp::uri::request::URI;
+use rtsp::uri::Host;
```
Comment (sgodwincs) [inline examples/client.rs]: Can you combine these two imports:

```
use rtsp::uri::{RTSP_DEFAULT_PORT, Host};
```

Comment (ultrasaurus) [reply examples/client.rs]: nice!

**Comment by ultrasaurus** [comment]
feedback addressed 🦈

**Comment by sgodwincs** [comment]
Thanks!

**Pull request merged**